### PR TITLE
Shank 328 status artifact when custom cluster

### PR DESCRIPTION
**Pull request by eclendenning-va**
https://vasdvp.atlassian.net/browse/SHANK-328

https://tools.health.dev-developer.va.gov/jenkins/job/department-of-veterans-affairs/job/health-apis-deployer/job/SHANK-328-status-artifact-when-custom-cluster/5/artifact/status.us-gov-west-1a.json

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -41,7 +41,7 @@ export AWS_DEFAULT_REGION=us-gov-west-1
 test -n "$ENVIRONMENT"
 test -f "$WORKSPACE/environments/$ENVIRONMENT.conf"
 . "$WORKSPACE/environments/$ENVIRONMENT.conf"
-DEFAULT_CLUSTER_ID=$CLUSTER_ID
+export DEFAULT_CLUSTER_ID=$CLUSTER_ID
```
Comment (jhulbert-va) [inline build.sh]: Should `/bin/deployment-status` be updated to just receive the cluster-id as `$1`? Making this global doesn't really buy us anything more than that would.

Comment (jhulbert-va) [reply build.sh]: Side Note: What other scripts use this?

Comment (eclendenning-va) [reply build.sh]: Only build.sh

File: bin/deployment-status [hunk 2]
```diff
@@ -2,20 +2,22 @@
 set -euo pipefail
 if [ "${DEBUG:-false}" == true ]; then set -x; fi
 
+DEFAULT_CLUSTER_ID=$1
```
Comment (jhulbert-va) [inline bin/deployment-status]: This should just be `CLUSTER_ID` or `STATUS_CLUSTER_ID` deployment status shouldn't know whether its the default or not, just that it needs to get the status for that ID.

**Pull request merged**